tests/acceptance/cancelBuilds.sh

```diff
@@ -0,0 +1,21 @@
+SCRIPT_PATH=$(dirname "$0")
+SCRIPT_PATH=$( cd "${SCRIPT_PATH}" && pwd )  # normalized and made absolute
+
```

Comment:
It would be good to add comments that describe what the text typically looks like in recentBuilds.txt and thisBuildInfo.txt

Future readers will find it almost impossible to reverse-engineer what all the following `awk` commands are processing.

Reply:
I have added some comments now. I hope it is understandable now